lib/daru/clean/clean.rb

```diff
@@ -0,0 +1,76 @@
+module Daru
+  # Module encapsulating advanced data cleaning methods on DataFrame.
+  module Clean
+    # replace elements with those from given set of elements
+    # based on similarity
```

Comment:
1. What "similarity" is in this context?
2. I don't believe that method name properly reflects what it does (the name suggests something is _matched_ -- probably returning "match result", while in example/docs values are _replaced_).

Reply:
1. By similarity, it means the Levenshtein distance between the strings as mentioned [here](https://github.com/seamusabshere/fuzzy_match#top-3-reasons-you-should-use-fuzzymatch).
2. I will rename it to fuzzy_replace to make it more intuitive. We can perhaps add a non-destructive version.